ui/src/apps/clusterinfo/components/component_panel.js

```diff
@@ -0,0 +1,28 @@
+import { Col, Row } from 'antd';
+import React from 'react';
+import styles from './component_panel.module.less';
+import alive_dead_cnt from './utils';
```

Comment:
This is not the naming convension we follow. Please use camelCase and full name.

Reply:
Done